borges/templates/consumer-deployment.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-consumer
+  labels:
+    app: {{ template "name" . }}-consumer
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "name" . }}-consumer
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - borges
+            - consumer
+            - --workers={{ .Values.consumer.workers }}
+            - --loglevel={{ .Values.consumer.loglevel }}
+            - --timeout={{ .Values.consumer.timeout }}
+            {{ if .Values.consumer.profiler }}
+            - --profiler
+            - --profiler-port={{ .Values.consumer.profilerPort }}
+            {{ end }}
+            {{ if .Values.consumer.metrics }}
+            - --metrics
+            - --metrics-port={{ .Values.consumer.metricsPort }}
+            {{ end }}
+
+          ports:
+            {{ if .Values.consumer.profiler }}
+            - containerPort: {{ .Values.consumer.profilerPort }}
+            {{ end }}
+            {{ if .Values.consumer.metrics }}
+            - containerPort: {{ .Values.consumer.metricsPort }}
+            {{ end }}
+          env:
```

Comment:
Just to have in mind, we are going to need a new environment variable in the near future to bucketize the borges output: https://github.com/src-d/borges/pull/184/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59

Reply:
What consequences does it have in how the program is started?

Reply:
Ok, I guess we can just add it when the time comes. Feel free to create a PR 😍

Reply:
perfect! we'll do that!